beniget/beniget.py

```diff
@@ -257,7 +257,9 @@ def defs(self, node):
         stars = []
         for d in reversed(self._definitions):
             if name in d:
-                return d[name] if not stars else stars + d[name]
+                # have to make sure stars and d[name] are lists otherwise
+                # they can't be concatenated
+                return d[name] if not stars else list(stars) + list(d[name])
```

Comment:
stars is always a list, so I think only the second cast is needed.

I'd also appreciate a test case :-)